frame/tips/src/benchmarking.rs

```diff
@@ -164,14 +177,62 @@ benchmarks! {
 		let reason_hash = T::Hashing::hash(&reason[..]);
 		let hash = T::Hashing::hash_of(&(&reason_hash, &beneficiary));
 		ensure!(Tips::<T>::contains_key(hash), "tip does not exist");
+
+		println!(
+			"Benchmark-close-tip-hash-{:#?}",
+			hash,
+		);
+
 		create_tips::<T>(t, hash.clone(), value)?;
 
 		let caller = account("caller", t, SEED);
 		// Whitelist caller account from further DB operations.
 		let caller_key = frame_system::Account::<T>::hashed_key_for(&caller);
 		frame_benchmarking::benchmarking::add_to_whitelist(caller_key.into());
+
+		println!(
+			"Benchmark-close-tip-exit",
+		);
 	}: _(RawOrigin::Signed(caller), hash)
 
+	slash_tip {
+		let t in 1 .. MAX_TIPPERS;
+
+		println!(
+			"Benchmark-slash-tip-entry",
+		);
+
+		// Make sure pot is funded
+		setup_pod_account::<T>();
+
+		// Set up a new tip proposal
+		let (member, reason, beneficiary, value) = setup_tip::<T>(0, t)?;
+		let value = T::Currency::minimum_balance().saturating_mul(100u32.into());
+		TipsMod::<T>::tip_new(
+			RawOrigin::Signed(member).into(),
+			reason.clone(),
+			beneficiary.clone(),
+			value
+		)?;
+
+		let reason_hash = T::Hashing::hash(&reason[..]);
+		let hash = T::Hashing::hash_of(&(&reason_hash, &beneficiary));
+		ensure!(Tips::<T>::contains_key(hash), "tip does not exist");
+
+		println!(
+			"Benchmark-slash-tip-hash-{:#?}",
+			hash,
+		);
+
+		TipsMod::<T>::slash_tip(
+			RawOrigin::Root.into(),
+			hash.clone(),
+		)?;
```

Comment:
Why are you calling slash tip here?

It is called below already when you do benchmarking

Reply:
Just delete this and it should work fine.